Client/Rewards/AdsMediaReporting.swift

```diff
@@ -8,7 +8,7 @@ import BraveRewards
 
 class AdsMediaReporting: TabContentScript {
     let rewards: BraveRewards
-    let tab: Tab
+    weak var tab: Tab?
```

Comment:
Can you sneak in the same change in `RewardsReporting.swift` :) I missed that here and this made me realize that